dex/networks/eth/params.go

```diff
@@ -5,10 +5,31 @@ package eth
 
 import "decred.org/dcrdex/dex"
 
-var UnitInfo = dex.UnitInfo{
-	AtomicUnit: "gwei",
-	Conventional: dex.Denomination{
-		Unit:             "ETH",
-		ConversionFactor: 1e9,
-	},
-}
+const (
+	// GweiFactor is the amount of wei in one gwei. Eth balances are floored
+	// as gwei, or 1e9 wei. This is used in factoring.
+	GweiFactor = 1e9
+	// InitGas is the amount of gas needed to initialize an ethereum swap.
+	//
+	// The price of a normal transaction is 21000 gas. However, when
+	// contract methods are called it is difficult to descern where gas is
+	// used. The formal declaration of what costs how much can be found at
+	// https://ethereum.github.io/yellowpaper/paper.pdf in Appendix G.
+	// The current value here is an appoximation based on tests.
+	//
+	// TODO: When the contract is solidified, break down evm functions
+	// called and the gas used for each. (◍•﹏•)
+	InitGas = 158000 // gas
+
+	RedeemGas = 63000 // gas
```

Comment:
Since the contracts are versioned now, these gas estimates will also need to be versioned.